lib/db/meta.go

```diff
@@ -392,7 +396,10 @@ func (m *countsMap) devices() []protocol.DeviceID {
 
 	for _, dev := range m.counts.Counts {
 		if dev.Sequence > 0 {
-			id := protocol.DeviceIDFromBytes(dev.DeviceID)
+			id, err := protocol.DeviceIDFromBytes(dev.DeviceID)
+			if err != nil {
+				panic(err)
```

Comment:
This "should never happen" I guess.  Unless there is database corruption or some piece of code put garbage in these DB entries.  IMO panicking is almost never the right thing to do, but I don't know this part of the code well enough to propose a better approach.